beziers/path/__init__.py

```diff
@@ -92,36 +92,36 @@ def fromNodelist(klass, array):
     return self
 
   @classmethod
-  def fromFonttoolsGlyph(klass, glyph, glyphset):
+  def fromFonttoolsGlyph(klass, glyph, glyphset, glyfTable):
```

Comment:
this method won't work for CFF glyphs.
I recommed that you use instead the generic _TTGlyphSet/_TTGlyph interface defined in fontTools.ttLib.ttFont module.
You get a glyph set object with TTFont.getGlyphSet() method, which behaves like a dict of glyph objects, which have a draw() method that takes a fonttools segment pen. Things like horizontal/vertical metrics or the lsb offset (for TT glyf) are handled there automatically

Reply:
In fact, I would be happiest with an interface something like `BezierPath.fromFonttoolsGlyph(font, "Aacute")`, hiding all the fonttools internals from the user altogether.